content/cloud-docs/api-docs/teams.mdx

```diff
@@ -77,6 +77,8 @@ $ curl \
 
 ### Sample Response
 
+The `sso-team-id` attribute is only returned if this team's organization is in the Terraform Cloud Business tier.
```

Comment:
I left a comment on [this](https://github.com/hashicorp/terraform-website/pull/2221) PR but will add it here as well. I imagine we'll want to mention here (and elsewhere that similar text has been added) that this is also going to exist in enterprise and potentially also add text that specifies what version the field will exist in there (e.g., `This attribute is also available in Terraform Enterprise as of version 202204-1` or something similar).

Reply:
+1  But just to check - is the version we're listing already available? If not, I think we should wait until it is available before adding a future-facing statement.

"The `sso-team-id` attribute is only returned of this team's organization is in the Terraform Cloud Business tier or you set it in Terraform Enterprise version 202204-1 and later."

Because you have to have set it right? (I think that's what I read) I just don't want folks to think it'll always be there in TFE if it won't be.

Reply:
> is the version we're listing already available? If not, I think we should wait until it is available before adding a future-facing statement.

It is not! We won't merge any docs changes (this one or #2221 ) until the changes are actually released.

> Because you have to have set it right? (I think that's what I read) I just don't want folks to think it'll always be there in TFE if it won't be.

You do need to set the value, yes. I will double check if the API returns the empty field (`sso-team-id : ""`) if you dont have it set, or if SSO isnt enabled...